Content.Client/Ghost/UI/GhostGui.cs

```diff
@@ -15,16 +13,21 @@ public sealed class GhostGui : Control
         private readonly Button _returnToBody = new() {Text = Loc.GetString("ghost-gui-return-to-body-button") };
         private readonly Button _ghostWarp = new() {Text = Loc.GetString("ghost-gui-ghost-warp-button") };
         private readonly Button _ghostRoles = new();
-        private readonly GhostComponent _owner;
+        private readonly Button _ghostToggleGhosts = new() {Text = Loc.GetString("ghost-gui-toggleghosts", ("switch", Loc.GetString("ghost-gui-no"))) };
+        private readonly Button _ghostToggleFOV = new() {Text = Loc.GetString("ghost-gui-togglefov", ("switch", Loc.GetString("ghost-gui-yes"))), Pressed = true };
+        private readonly Button _ghostToggleShadows = new() {Text = Loc.GetString("ghost-gui-toggleshadows", ("switch", Loc.GetString("ghost-gui-no"))) };
+        private readonly Button _ghostToggleLights = new() {Text = Loc.GetString("ghost-gui-togglelight", ("switch", Loc.GetString("ghost-gui-no"))) };
```

Comment:
i think these should be ghost actions and not hardcoded into the ghost gui. i believe macoron was doing nightvision as an action so could check his PR? its not too hard to make these actions regardless, feel free to ask if you need help w/ that